guides/common/modules/con_acd_introduction.adoc

```diff
@@ -0,0 +1,113 @@
+Now, the ACD approach presents a new idea: deploying hosts to run an application that requires a predefined set of services.
+
+Generally, an application consists of multiple services and therefore requires different kinds of hosts connected to each other.
+These hosts have dependencies on each other, for example, an application server might require a database server.
+
+The number of deployed hosts depends on the expected usage of the application.
+You can scale an application instance vertically via compute profile or horizontally by choosing multiple hosts to run the same service to handle increased workloads.
+
+Within ACD, Ansible playbooks are executed on the {ProjectServer}.
+Ensure that {ProjectServer} can connect to all managed hosts using Ansible.
+
+.Host Centric vs. Application Centric Deployment
+[caption=]
+[cols="20%,40%,40%"]
+|===
+|
+|Host Centric Deployment
+|Application Centric Deployment
+
+| Requirements
+| none (possible by default)
+| ACD plugin
+
+|Focus
+|individual hosts
+|applications
+
+|Number of Hosts
+|1
+|1 to many
+
+|Automation
+|via Host Group
+|via Ansible playbook and application definition
+
+|Advantages
+|full control over each host
+|deploy a number of hosts at once; self service for technically less inclined; scale vertically and/or horizontally
+
+|Disadvantages
+|extra abstraction layer between a host and its purpose
+|dependency on/limitation to existing application templates
+|===
+
+[id="{context}_usage_example"]
+== Usage Example
+
+The following application centric deployment example illustrates the deployment of hosts in an application centric approach.
+The web application requires an HAProxy, an Apache web server, a Tomcat application server, and a database server.
+
+Using ACD, this web application can be deployed and configured in a single step:
+
+. Import the Ansible playbook
++
+Upload the Ansible playbook to your {ProjectServer}.
+Specify the path and the name of the Ansible playbook via management UI.
+. Create an application definition
++
+This example requires four different services: proxy servers, web servers, application servers, and database servers.
+Those hosts are ideally part of a host group.
+. Optional: Customize Ansible variables
++
+You can either use the default values, change values, or lock the Ansible variables from being edited for an application instance.
+Ansible variables are read from the group variables of the Ansible playbook.
+. Optional: Overwrite variables from the host group
++
+For example, change the compute profile or set a different lifecycle environment.
+. Deploy an application instance
++
+** You may simply deploy the application instance;
+** run the Ansible playbook as is;
+** or customize the Ansible playbook before running it.
+
+[NOTE]
+====
+If you want to install a package using an Ansible playbook and the host is subscribed to {ProjectServer}, ensure that the package exists in the content view.
```

Comment:
Content View is capitalised in Foreman documentation.

Reply:
Fixed.